src/transformers/models/auto/modeling_auto.py

```diff
@@ -600,6 +600,12 @@
     ]
 )
 
+MODEL_FOR_DOCUMENT_QUESTION_ANSWERING_MAPPING_NAMES = OrderedDict(
+    [
+        ("layoutlm", "LayoutLMForQuestionAnswering"),
```

Comment:
@NielsRogge @Narsil here's what I added. I can additionally add layoutlmv2 and v3 here (while keeping them in `MODEL_FOR_QUESTION_ANSWERING_MAPPING_NAMES`)? Or leave it as just LayoutLMv1 for now.

Reply:
You can add v2 and v3 here as well

Reply:
ok will do!